build.rs

```diff
@@ -0,0 +1,21 @@
+use std::{ env, fs };
+use fs_extra::{copy_items, dir::CopyOptions};
+
+fn copy_js_folder() {
+    let mut copy_options = CopyOptions::new();
+    let out_dir = env::var("OUT_DIR").unwrap();
+
+    copy_options.overwrite = true;
+    copy_options.copy_inside = true;
+
+    let mut docs_dir = out_dir;
+    docs_dir.push_str("/../../../../doc");
+
+    fs::create_dir_all(&docs_dir).unwrap();
+    copy_items(&vec!["./doc/js/mermaid.min.js"], &docs_dir, &copy_options).unwrap();
```

Comment:
1) There's no need to pass a `vec!` reference, since `copy_items` expects a reference to slice. You can construct slice here instead of vector: `copy_items(&["..."], ...)`

2) As far as I see there's no reason to use the `fs_extra` crate, [fs::copy]](https://doc.rust-lang.org/std/fs/fn.copy.html) should suffice

Reply:
That's what I though as well! The `fs_extra`function  has been replaced by the `std::fs::copy`. I also renamed the function to something more specific. I think it's good enough for the moment.